### ISSUE TITLE
SystemCheckError: System check identified some issues

### ISSUE THREAD

**Issue by maoaiz** [nephila/djangocms-installer#278]
I'm trying to install Django CMS using djangocms-installer==0.8.11.

I'm using virtualenv, I've created a enviroment and I've installed the installer using `pip install djangocms-installer`. If I run `djangocms -p /my/path my_cms` according the [install section](https://www.django-cms.org/en/#install) in the Django CMS web, I have the next error:

```sh
Database configuration (in URL format) [default sqlite://localhost/project.db]: 
django CMS version (choices: 2.4, 3.0, 3.1, 3.2, 3.3, stable, rc, develop) [default stable]: 
Django version (choices: 1.4, 1.5, 1.6, 1.7, 1.8, 1.9, stable) [default stable]: 
Activate Django I18N / L10N setting; this is automatically activated if more than language is provided (choices: yes, no) [default yes]: 
Install and configure reversion support (choices: yes, no) [default yes]: 
Languages to enable. Option can be provided multiple times, or as a comma separated list. Only language codes supported by Django can be used here: es
Optional default time zone [default America/Bogota]: 
Activate Django timezone support (choices: yes, no) [default yes]: 
Activate CMS permission management (choices: yes, no) [default yes]: 
Use Twitter Bootstrap Theme (choices: yes, no) [default no]: yes
Use custom template set [default no]: 
Load a starting page with examples after installation (english language only). Choose "no" if you use a custom template set. (choices: yes, no) [default no]:  
Creating the project
Please wait while I install dependencies
Dependencies installed
Creating the project
SystemCheckError: System check identified some issues:

ERRORS:
djangocms_video.VideoPlayer.poster: (fields.E300) Field defines a relation with model 'Image', which is either not installed, or is abstract.
djangocms_video.VideoSource.source_file: (fields.E300) Field defines a relation with model 'File', which is either not installed, or is abstract.
djangocms_video.VideoTrack.src: (fields.E300) Field defines a relation with model 'File', which is either not installed, or is abstract.
The installation failed.
Do you want to clean up by removing /my/path/cms?
	Warning: this will delete all files in:
		/my/path/cms
Do you want to cleanup? [y/N]
```

**Comment by maoaiz** [comment]
According [the documentation](http://docs.django-cms.org/en/release-3.3.x/introduction/install.html) realize that is necesary install django-filler using `-f` option. I think that the error message is not clear.

**Comment by yakky** [comment]
@maoaiz actually the error you reported is due some changes occurring in djangocms-video
They will be addressed in upcoming version 0.8.12

**Comment by yakky** [comment]
Fixed in b193de81b609c048a99c0fa2a2b420e1084398e8
@maoaiz thanks for reporting this
